charts/studio/templates/deployment-studio-ui.yaml

```diff
@@ -33,6 +33,8 @@ spec:
       serviceAccountName: {{ include "studio.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.studioUi.podSecurityContext | nindent 8 }}
+        fsGroup: 103 # dvc
+        fsGroupChangePolicy: "OnRootMismatch"
```

Comment:
UI is not using this user. AFAIR `1001` ?

Reply:
UI doesn't use blobvault volume so it's fine :+1:

Reply:
Fixed